handlers/base.go

```diff
@@ -0,0 +1,69 @@
+package handlers
+
+import (
+	"database/sql"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/joho/godotenv"
+	conn "github.com/somos831/somos-backend/db"
+	"github.com/somos831/somos-backend/validators"
+)
+
+type Server struct {
+	db        *sql.DB
+	Router    *http.ServeMux
+	Validator validators.Validator
+}
+
+func (server *Server) Initialize() {
```

Comment:
We could rename this method to `NewServer` or `InitServer` to follow [go naming conventions](https://pkg.go.dev/net/http#NewRequest).